templates/case/hearing/index.html.twig

```diff
@@ -91,17 +93,27 @@
                                 <div class="row">
                                     <div class="col">
                                         <h2 class="m-0 p-0">
-                                            {% if not post.getForwardedOn %}
+                                            {% if post.forwardedOn is defined and not post.getForwardedOn %}
+                                                <span class="badge badge-pill badge-warning">N</span>
```

Comment:
What does “N” stand for? “No”? Should it be translated?

Reply:
"N" indicates something new and/or something that needs processing.